app/controllers/concerns/idv_step_concern.rb

```diff
@@ -93,6 +93,14 @@ def extra_analytics_properties
     extra
   end
 
+  def opt_in_analytics_properties
+    extra = {}
+    if IdentityConfig.store.in_person_proofing_opt_in_enabled
+      extra = { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
+    end
+    extra
```

Comment:
I bet we could remove the variable assignment and simplify
```suggestion
    if IdentityConfig.store.in_person_proofing_opt_in_enabled
      { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
    else
      {}
    end
```